### PR TITLE
[gitops-wg] Add Akuity members

### PR DESCRIPTION
**Pull request by wojtekidd**
### Consent to be added:

- [x] Hong Wang @wanghong230
- [x] Jesse Suen @jessesuen
- [x] Wojtek Cichoń @wojtekidd
- [x] Daniel Hoang @dhpup
- [x] Remington Breeze @rbreeze
- [x] Yuan Tang @terrytangyuan

### CODE REVIEW

**Review by scottrigby — CHANGES_REQUESTED** [review]
hi @wojtekidd 👋 two things:

1. We haven't yet merged a PR adding interest on behalf of others. Would you please ask Hong, Jesse, Daniel, Remington, and Yuan to comment here that they would like to be added, then we can merge this? It's just good practice for each person to agree themselves 😄 
2. Would you please also add your GitHub username(s) as well, now that we have that column?

File: gitops-wg/interested-parties.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 Organization | Person | GitHub Username
 -- | -- | --
 Aivitex | Lothar Schulz
+Akuity | Hong Wang
+Akuity | Jesse Suen
+Akuity | Wojtek Cichoń
+Akuity | Daniel Hoang
+Akuity | Remington Breeze
+Akuity | Yuan Tang
```
Comment (scottrigby) [inline gitops-wg/interested-parties.md:11]: I was able to find the correct GitHub usernames for everyone but Daniel Hoang.

Everyone added here @wanghong230, @jessesuen, @rbreeze, @terrytangyuan please add your consent (thumbs up, comment, etc), and if you know Daniel's GitHub username we can ping them as well.

```suggestion
Akuity | Hong Wang | wanghong230
Akuity | Jesse Suen | jessesuen
Akuity | Wojtek Cichoń | wojtekidd
Akuity | Daniel Hoang
Akuity | Remington Breeze | rbreeze
Akuity | Yuan Tang | terrytangyuan
```

Comment (dhpup) [reply gitops-wg/interested-parties.md:11]: My username is dhpup

**Comment by terrytangyuan** [comment]
Yes, thanks for adding this!

**Comment by wanghong230** [comment]
Yes, thanks for adding this!

**Comment by rbreeze** [comment]
Yes, thank you!!

**Comment by todaywasawesome** [comment]
We're only missing @jessesuen

File: gitops-wg/interested-parties.md [hunk 2]
```diff
@@ -54,8 +60,6 @@ Intuit | Edward Lee
 Intuit | Henrik Blixt
 Intuit | Alexander Matyushentsev
```
Comment (terrytangyuan) [inline gitops-wg/interested-parties.md:55]: @wojtekidd Also need to update Alex's entry. cc @alexmt

**Comment by jessesuen** [comment]
I give consent. Thank you!

File: gitops-wg/interested-parties.md (hunk 2 above)
Comment (alexmt) [reply gitops-wg/interested-parties.md:55]: Yes please

Comment (wojtekidd) [reply gitops-wg/interested-parties.md:55]: Done

**Comment by wojtekidd** [comment]
Should be OK now @scottrigby

**Review by todaywasawesome — APPROVED** [review]
LGTM

**Review by alexmt — APPROVED** [review]

**Review by scottrigby — APPROVED** [review]

**Pull request merged**